bluebottle/payments/services.py

```diff
@@ -1,6 +1,6 @@
 from bluebottle.payments.exception import PaymentException
 import re
-from django.conf import settings
+from bluebottle.clients import properties
```

Comment:
What I got from Ivo is that injecting properties into your settings isn't good. 
@iivvoo could you shed some light on this?